sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/EventRoutesSyncSamples.java

```diff
@@ -0,0 +1,119 @@
+import com.azure.digitaltwins.core.implementation.models.ErrorResponseException;
+import com.azure.digitaltwins.core.models.EventRoute;
+import com.azure.identity.ClientSecretCredentialBuilder;
+
+import java.util.UUID;
+
+/**
+ * This sample will list all the current event routes in your Azure Digital Twins instance,
+ * then it will retrieve a particular event route. If the parameter for an existing event
+ * route endpoint name is provided then it will also create a new event route, and it will
+ * delete that newly created event route.
+ */
+public class EventRoutesSyncSamples {
+    private static DigitalTwinsClient client;
+
+    public static void main(String[] args) {
+        SamplesArguments parsedArguments = new SamplesArguments(args);
+
+        client = new DigitalTwinsClientBuilder()
+            .credential(
+                new ClientSecretCredentialBuilder()
+                    .tenantId(parsedArguments.getTenantId())
+                    .clientId(parsedArguments.getClientId())
+                    .clientSecret(parsedArguments.getClientSecret())
+                    .build()
+            )
+            .endpoint(parsedArguments.getDigitalTwinEndpoint())
+            .httpLogOptions(
+                new HttpLogOptions()
+                    .setLogLevel(parsedArguments.getHttpLogDetailLevel()))
+            .buildClient();
+
+        ConsoleLogger.printHeader("List event routes");
+        ConsoleLogger.print("Listing all current event routes in your Azure Digital Twins instance");
+        PagedIterable<EventRoute> eventRoutes = null;
+        try {
+            eventRoutes = client.listEventRoutes();
+        } catch (ErrorResponseException ex) {
+            ConsoleLogger.printFatal("Failed to list event routes");
+            ex.printStackTrace();
+            System.exit(0);
+        }
+
+        String existingEventRouteId = null;
+        for (EventRoute eventRoute : eventRoutes) {
+            existingEventRouteId = eventRoute.getId();
+            ConsoleLogger.print(String.format("     EventRouteId: %s", eventRoute.getId()));
+            ConsoleLogger.print(String.format("     EventRouteEndpointName: %s", eventRoute.getEndpointName()));
+            if (eventRoute.getFilter() != null)
+            {
+                ConsoleLogger.print(String.format("     Filter: %s", eventRoute.getFilter()));
+            }
+            ConsoleLogger.print("");
+        }
+
+        if (existingEventRouteId != null) {
+            ConsoleLogger.printHeader("Get event route");
+            ConsoleLogger.print(String.format("Getting a single event route with Id %s", existingEventRouteId));
+            try {
+                EventRoute existingEventRoute = client.getEventRoute(existingEventRouteId);
+                ConsoleLogger.print(String.format("Successfully retrieved event route with Id %s", existingEventRouteId));
+                ConsoleLogger.print(String.format("     EventRouteId: %s", existingEventRoute.getId()));
```

Comment:
why the empty spaces? if you intend to tab in , use \t ?

Reply:
Ah, good thinking. I'll just \t instead